src/Evolution/Systems/Cce/Tags.hpp

```diff
@@ -0,0 +1,232 @@
+/// derivative of \f$J\f$ at fixed compactified radius \f$y\f$ (to be contrasted
+/// with the physical Bondi radius, which is not directly used for numerical
+/// grids).
+struct BondiH : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 2>>;
+  static std::string name() noexcept { return "H"; }
+};
+
+/// Bondi parameter \f$J\f$
+struct BondiJ : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 2>>;
+  static std::string name() noexcept { return "J"; }
+};
+
+/// Bondi parameter \f$\bar{J}\f$
+struct BondiJbar : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, -2>>;
+  static std::string name() noexcept { return "Jbar"; }
+};
+
+/// Bondi parameter \f$K = \sqrt{1 + J \bar{J}}\f$
+struct BondiK : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 0>>;
+  static std::string name() noexcept { return "K"; }
+};
+
+/// Bondi parameter \f$Q\f$
+struct BondiQ : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 1>>;
+  static std::string name() noexcept { return "Q"; }
+};
+
+/// Bondi parameter \f$\bar{Q}\f$
+struct BondiQbar : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, -1>>;
+  static std::string name() noexcept { return "Qbar"; }
+};
+
+/// Bondi parameter \f$U\f$
+struct BondiU : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 1>>;
+  static std::string name() noexcept { return "U"; }
+};
+
+/// Bondi parameter \f$\bar{U}\f$
+struct BondiUbar : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, -1>>;
+  static std::string name() noexcept { return "Ubar"; }
+};
+
+/// Bondi parameter \f$U\f$
+struct BondiW : db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, 0>>;
+  static std::string name() noexcept { return "W"; }
+};
+
+/// The derivative with respect to the numerical coordinate \f$y = 1 - 2R/r\f$
+template <typename Tag>
+struct Dy : db::PrefixTag, db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, Tag::type::type::spin>>;
+  using tag = Tag;
+  static const size_t dimension_to_differentiate = 2;
+  static const size_t dim = 3;
+  static std::string name() noexcept { return "Dy(" + Tag::name() + ")"; }
+};
+
+// prefix tags associated with the integrands which are used as input to solvers
+// for the CCE equations
+
+/// A prefix tag representing a quantity that will appear on the right-hand side
+/// of an explicitly regular differential equation
+template <typename Tag>
+struct Integrand : db::PrefixTag, db::SimpleTag {
+  using type = Scalar<SpinWeighted<ComplexDataVector, Tag::type::type::spin>>;
+  using tag = Tag;
+  static std::string name() noexcept {
+    return "Integrand(" + Tag::name() + ")";
+  }
+};
+
+/// A prefix tag representing the boundary data for a quantity on the extraction
+/// surface.
+template <typename Tag>
+struct BoundaryValue : db::PrefixTag, db::SimpleTag {
```

Comment:
You can remove the `Value` suffix. I leave it up to you to decide if you want to do this or not :)

Reply:
If you don't feel strongly about it, I think I'll leave it as is, if only because I'd like it to carry the connotation of being used during integral evaluations, for which I think it is more common to use the phrase 'the boundary value' for the definite integration.